src/styles/main.css

```
@@ -134,24 +152,39 @@ footer.footer p:not(:last-child) {
+  gap: 12px;
   max-width: 1280px;
-  height: 76px;
+  height: 90px;
   margin: 0 auto;
-  padding: 20px 54px;
+  padding: 14px 45px 8px;
 }
 
 .logo {
-  position: absolute;
+}
```

Stylelint (17.7.0)

[error] Empty block (block-no-empty)

(block-no-empty)